timemachine/fe/mle.py

```diff
@@ -276,12 +281,12 @@ def infer_node_vals_and_errs_networkx(
             ref_node_vals.append(d[ref_node_val_prop])
             ref_node_stddevs.append(d.get(ref_node_stddev_prop, 0.0))
 
-    edge_idxs = np.array(sg_relabeled.edges)
-
+    edges = np.array(sg_relabeled.edges)
+    edge_idxs = np.array([e[:2] for e in edges])  # remove edge key in MultiDiGraph if present
```

Comment:
nit: avoid array -> list -> array
```suggestion
    edge_idxs = edges[:, :2]  # remove edge key in MultiDiGraph if present
```